src/cmd_parser.c

```diff
@@ -133,7 +133,9 @@ static GOptionEntry cmd_entries[] =
     { "simple-md-filenames", 0, 0, G_OPTION_ARG_NONE, &(_cmd_options.simple_md_filenames),
       "Do not include the file's checksum in the metadata filename.", NULL },
     { "retain-old-md", 0, 0, G_OPTION_ARG_INT, &(_cmd_options.retain_old),
-      "Keep around the latest (by timestamp) N copies of the old repodata.", "NUM" },
+      "Specify NUM to 0 to remove all repodata present in old repomd.xml or any other positive number to keep all old repodata. "
+      "Use --compatibily flag to get the behavior of original createrepo: "
```

Comment:
`--compatibily` -> `--compatibility`

Reply:
Thank you! I have updated the PR, apparently I can't write..